api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/redis/BaseRedisProperties.java

```diff
@@ -55,9 +56,10 @@
     private int port = 6379;
 
     /**
-     * Connection timeout in milliseconds.
+     * Command timeout, default from Lettuce is 60s.
      */
-    private int timeout = 2000;
+    @DurationCapable
+    private String timeout;
```

Comment:
I think it might be good to assign a default value here, even if the default is picked up from somewhere else if left unassigned. This will show up in the docs and makes it easy for someone to figure out what the actual value is without having to parse code, and will also help drive the configuration metadata.